opentelemetry-api/src/global/error_handler.rs

```diff
@@ -1,6 +1,8 @@
 use std::sync::PoisonError;
```

Comment:
I think similar to `GlobalTracerProvider`. We need a `GlobalLogEmitterProvider` here

Reply:
Yes. However, that would also mean that we would have to define `opentelemetry-api` traits for a `GlobalLogEmitterProvider`, while the spec explicitly leaves out an API, as the SDK is meant to be utilized only by library developers. 

Although it doesn't sound very idiomatic, what do you think about having a `GlobalLogEmitterProvider` in `opentelemetry-sdk`, in the `logs` module?

Reply:
Yeah I think that works as the user doesn't suppose to interact with `GlobalLogEmitterProvider`